libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -9,17 +9,45 @@ import {
   KeyValueTable,
   KeyValueTableRow,
   AccordionPanel,
+  AsyncRenderer,
+  Splash,
 } from '@vegaprotocol/ui-toolkit';
 import startCase from 'lodash/startCase';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
-import type { DealTicketQuery_market } from './__generated__/DealTicketQuery';
+import type {
+  DealTicketQuery,
+  DealTicketQuery_market,
+} from './__generated__/DealTicketQuery';
 import BigNumber from 'bignumber.js';
+import { useQuery } from '@apollo/client';
+import { DEAL_TICKET_QUERY } from './deal-ticket-container';
```

Comment:
chore (blocking): Lets not share the query. Instead can we update the deal ticket query to only contain what it needs and add a new market info query for everything it needs.